package.json

```diff
@@ -4,7 +4,9 @@
   "description": "RESTful API for a forum site.",
   "main": "server.ts",
   "scripts": {
-    "test": "mocha --require ts-node/register --require ./test/global-fixtures.ts ./test/* --exit",
+    "test": "TESTING=true mocha --require ts-node/register --require ./test/global-fixtures.ts ./test/* --colors --exit",
```

Comment:
we should use NODE_ENV as the environment variable. Also please use cross-env or similar so it's compatible on all platforms